spec/migrations/20151026152536_remove_automate_worker_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "spec_helper"
+require_migration
+
+describe RemoveAutomateWorker do
+  let(:miq_worker_stub) { migration_stub(:MiqWorker) }
+
+  migration_context :up do
+    it "up" do
+      miq_worker_stub.create!(:type => 'MiqAutomateWorker')
```

Comment:
Add a second worker here of a different type and verify that the migration does not kill that other worker type.